composer.json

```diff
@@ -43,7 +43,7 @@
     "require-dev": {
         "cakephp/cakephp": ">=3.2",
         "cakephp/cakephp-codesniffer": "2.*",
-        "phpunit/phpunit": "4.1.*"
+        "phpunit/phpunit": "5.7.*"
```

Comment:
Changing this to `"<6.0"` is better, that way composer can pick appropriate version based on php version. Using `"<6.0"` has worked fine on other plugins i maintain.